src/modules/SpeedButton.ts

```diff
@@ -11,8 +11,6 @@ export class SpeedButton extends MainButton {
     speedSettings = [0.5, 0.75, 1, 1.25, 1.5, 1.75, 2];
     speedSelectId = 'webriceSelect';
     speedElementClass = 'webriceSpeedElement';
-    // eslint-disable-next-line no-invalid-this
-    hideSpeedModule = this.closeUnusedSpeedComponents.bind(this);
```

Comment:
removing this and binding directly in the eventlistener reintroduces the run away eventlistener bug again fixed by #29